app/src/org/commcare/dalvik/activities/EntitySelectActivity.java

```diff
@@ -719,6 +719,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case MENU_ACTION:
                 triggerDetailAction();
                 return true;
+            // this is needed because superclasses do not implement the menu_settings click
+            case R.id.menu_settings:

```

Comment:
Any reason that this doesn't follow the convention of defining a local constant such as 'MENU_FOO = Menu.FIRST + N' and using it instead of R.id.menu_settings?